modules/metrics/src/main/java/com/spotify/apollo/metrics/FfwdConfig.java

```diff
@@ -102,6 +111,14 @@ int getInterval() {
       host.ifPresent(builder::host);
       port.ifPresent(builder::port);
 
+      if (flush) {
+        return () -> {
+          final FastForwardReporter reporter = builder.build();
+          reporter.start();
+          return reporter::stopWithFlush;
```

Comment:
How about moving the flag to this line, and merge this lambda with the one below for less code duplication?

```
return () -> {
        final FastForwardReporter reporter = builder.build();
        reporter.start();
        return flush ? reporter::stopWithFlush : reporter::stop;
      };
```